pages/api/predictions/index.js

```diff
@@ -2,7 +2,7 @@ export default async function handler(req, res) {
   const response = await fetch("https://api.replicate.com/v1/predictions", {
     method: "POST",
     headers: {
-      Authorization: `Token ${process.env.REPLICATE_API_TOKEN}`,
+      "Authorization": `Token ${process.env.REPLICATE_API_TOKEN}`,
```

Comment:
I don't think the quotes are needed here. JavaScript object keys only need quotes if they contain characters like dashes.